07-sampling.Rmd

```diff
@@ -660,50 +676,106 @@ In both our tactile and our virtual sampling activities, we used sampling for th
 
-Ninth and lastly, the idea of **random sampling**. We say a sampling procedure is *random* if we sample randomly from the population in an unbiased fashion. In our sampling activities, this would correspond to sufficiently mixing the bowl before each use of the shovel. 
+We say a sample of $n$ balls extracted using our shovel is **representative** of the population if it's contents "roughly resemble" the contents of the bowl. If so, then the proportion of the shovel's balls that are red can **generalize** to the proportion of the bowl's $N$ = `r num_balls` balls that are red. Or expressed differently, $\widehat{p}$ is a "good guess" of $p$. Now say we cheated when using the shovel and removed a number of white balls in favor of red balls. Then this sample would be **biased** towards red balls, and thus the sample would no longer be representative of the bowl.
 
-Phew, that's a lot of new terminology and notation to learn! Let's put them all together to describe the paradigm of sampling.
+The fourth and final set of terms and notation relate to the goal of sampling:
 
-**In general:**
+1. One way to ensure that a sample is unbiased and representative of the population is by using **random sampling**
+1. **Inference** is the act of "making a guess" about some unknown. **Statistical inference** is the act of making a guess about a population using a sample.
 
-* If the sampling of a sample of size $n$ is done at **random**, then
-* the sample is **unbiased** and **representative** of the population of size $N$, thus
-* any result based on the sample can **generalize** to the population, thus
-* the point estimate is a **"good guess"** of the unknown population parameter, thus
-* instead of performing a census, we can **infer** about the population using sampling.
+In our case, since the `rep_sample_n()` function uses your computer's [random number generator](https://en.wikipedia.org/wiki/Random_number_generation), we were in fact performing **random sampling**.
 
-**Specific to our sampling activity:**
+Let's now put all four sets of terms and notation together, keeping our sampling activities in mind:
 
-* If we extract a sample of $n=50$ balls at **random**, in other words, we mix all of the equally sized balls before using the shovel, then
-* the contents of the shovel are an **unbiased representation** of the contents of the bowl's `r num_balls` balls, thus
-* any result based on the shovel's balls can **generalize** to the bowl, thus
-* the sample proportion $\widehat{p}$ of the $n=50$ balls in the shovel that are red is a **"good guess"** of the population proportion $p$ of the $N=`r num_balls`$ balls that are red, thus
-* instead of manually going over all `r num_balls` balls in the bowl, we can **infer** about the bowl using the shovel. 
+* Since we extracted a sample of $n$ = 50 balls at *random*, in other words we mixed all of the equally sized balls before using the shovel, then
```

Comment:
```suggestion
* Since we extracted a sample of $n$ = 50 balls at *random*, we mixed all of the equally sized balls before using the shovel, then
```